test/runnable/literal.d

```diff
@@ -174,16 +174,28 @@ void f13907_3(wchar[3] a) {}
 auto f13907_12(char[1]) { return 1; }
 auto f13907_12(char[2]) { return 2; }
 
+auto f13907_123(char[1]) { return 1; }
+auto f13907_123(char[2]) { return 2; }
+auto f13907_123(char[3]) { return 3; }
+auto f13907_123(const(char)[]) { return 0; }
+
 void test13907()
 {
     static assert(!__traits(compiles, { f13907_1("\U00010000"w); }));
     static assert(!__traits(compiles, { f13907_1("\U00010000" ); }));
     f13907_2("\U00010000"w);
     f13907_2("\U00010000");
-    static assert(!__traits(compiles, { f13907_3("\U00010000"w); }));
-    static assert(!__traits(compiles, { f13907_3("\U00010000" ); }));
+  //f13907_3("\U00010000"w);    // Re-enable implicit length extension, from issue 13999 (unlisted codegen bug)
```

Comment:
It happens in win32 platform, without any args. Test case:

```d
void f13907_3(wchar[3] a) {}
void main()
{
    f13907_3("\U00010000"w);
}
```

Note that, git-head fails to compile the code. To reproduce runtime segfault, you need to use 2.066 or cherry-pick this PR.